app/src/main/java/com/github/se/eventradar/model/OverviewViewModel.kt

```diff
@@ -23,56 +18,21 @@ class OverviewViewModel(db: FirebaseFirestore = Firebase.firestore) : ViewModel(
 
   fun onSearchQueryChanged(query: String) {
     _uiState.value = _uiState.value.copy(searchQuery = query)
-    filterToDos()
-  }
-
-  private fun filterToDos() {
-    val query = _uiState.value.searchQuery
-    val filteredToDos =
-        _uiState.value.toDoList.getAllTask.filter { it.title.contains(query, ignoreCase = true) }
-    _uiState.value =
-        _uiState.value.copy(
-            toDoList = _uiState.value.toDoList.copy(getFilteredTask = filteredToDos))
-  }
-
-  fun getToDos() {
-    taskRef
-        .get()
-        .addOnSuccessListener { result ->
-          val todos =
-              result.documents.map { document ->
-                ToDo(
-                    id = document.id,
-                    title = document.data?.get("name") as String,
-                    assigneeName = document.data?.get("assigneeName") as String,
-                    dueDate = getLocalDate(document.data?.get("dueDate") as String),
-                    location =
-                        getLocation(
-                            document.data?.get("location_name") as String,
-                            document.data?.get("location_lat") as Double,
-                            document.data?.get("location_lng") as Double,
-                        ),
-                    description = document.data?.get("description") as String,
-                    status = getToDoStatus(document.data?.get("status") as String))
-              }
-          _uiState.value =
-              _uiState.value.copy(
-                  toDoList = ToDoList(todos, todos, _uiState.value.toDoList.getTask))
-        }
-        .addOnFailureListener { exception ->
-          Log.d("OverviewViewModel", "Error getting documents: ", exception)
-        }
   }
 
   fun onTaskClicked(id: String) {
-    val toDo = _uiState.value.toDoList.getAllTask.find { it.id == id }
+    // Temporarily commented out as we transition from handling to-dos to handling events.
+
+    /*
+    val to_Do = _uiState.value.toDoList.getAllTask.find { it.id == id }
```

Comment:
nit: there the naming convention `to_Do` is not standard compared to what it was before (`toDo`).

Reply:
It is indeed a bit confusing. However, I would argue that in Map.kt the naming convention is still 'toDo' and has not been changed. So either way is fine with me but the names should be consistent between OverviewViewModel.kt and Map.kt. In this case, I would recommend leaving everything as toDo even if it might not be the cleanest. As said above, these blocks can also be deleted for now since we have them in a previous commit.